docker-tools/Dockerfile-from-conda

```diff
@@ -21,13 +21,11 @@ RUN conda install --quiet --yes \
 ARG TORNADO_VERSION
 RUN sh -c 'if [[ ! -z "$TORNADO_VERSION" ]]; then echo Installing old tornado $TORNADO_VERSION; conda install --quiet --yes tornado=$TORNADO_VERSION; conda clean -ay; fi'
 
-# Clean the conda environment
-#RUN conda clean -ay
-
 RUN python -c "import tornado; print('tornado version=' + tornado.version)"
 # Workaround, just calling `bokeh info` crashes
 RUN env BOKEH_RESOURCES=cdn bokeh info
 
-ENV BOKEH_EXAMPLE_DIR bokeh_examples
-COPY hello_world.py $BOKEH_EXAMPLE_DIR/hello_world.py
-COPY hello_nodejs.py $BOKEH_EXAMPLE_DIR/hello_nodejs.py
+# Add some bokeh examples to be run
+ENV BOKEH_EXAMPLE_DIR /root/bokeh_examples
+COPY examples/models $BOKEH_EXAMPLE_DIR/models
```

Comment:
As long as we are going to just copy directories wholesale, I would say lets either copy `plotting` instead or in addition to. The `models` examples are all "low level" and I would not want that to be anyone's introdction

Reply:
I mostly want the examples just to quickly check that everything in the container works, I don't think these dockerfiles are used by 'beginners', although they might not be used at all ;) The reason I didn't put `plotting` is that it is quite big (55MB)

Reply:
In that case I'd say we could just hand pick a few examples, but we can revisit this later